packages/common/src/hooks/purchaseContent/usePurchaseContentFormConfiguration.ts

```diff
@@ -47,16 +64,29 @@ export const usePurchaseContentFormConfiguration = ({
         customAmount
       })
 
-      dispatch(
-        startPurchaseContentFlow({
-          extraAmount,
-          extraAmountPreset: amountPreset,
-          contentId: track.track_id,
-          contentType: ContentType.TRACK
-        })
-      )
+      if (purchaseMethod === PurchaseMethod.CARD) {
+        dispatch(
```

Comment:
this seems to be doing the same as line 77 below

Reply:
yep, they are the same now but we will be splitting the purchase path here i think